src/containers/CoursesPanel/index.jsx

```diff
@@ -33,12 +34,16 @@ export const CoursesPanel = () => {
           <CourseFilterControls {...courseListData.filterOptions} />
         </div>
       </div>
-      {hasCourses
-        ? (
-          <CourseList {...courseListData} />
-        ) : (
-          <NoCoursesView />
-        )}
+      <PluginSlot
+        id="courselist"
+      >
+        { hasCourses && <CourseList {...courseListData} /> }
+      </PluginSlot>
+      <PluginSlot
+        id="nocoursesview"
+      >
+        { !hasCourses && <NoCoursesView /> }
+      </PluginSlot>
```

Comment:
```suggestion
    {hasCourses ? (
      <PluginSlot
        id="course_list"
      >
        <CourseList {...courseListData} />
      </PluginSlot>
    ) : (
      <PluginSlot
        id="no_courses_view"
      >
        <NoCoursesView />
      </PluginSlot>
    )}
```

I made a couple suggestions here. I think we should keep the ternary as it feels cleaner and better exemplifies what is happening in the code. That may be more of a nit.

But I also think we should make sure the boolean is checked _before_ rendering a PluginSlot. The way it is currently written will have React render the slots but they will do nothing. Which doesn't really negatively impact us other than being somewhat confusing code and snapshot.

Then another nit was making the id snake-case which is something we haven't talked a ton about but is a pattern we've been using up to this point.